### DISCUSSION TITLE
Can I create multiple files by list?

### DISCUSSION THREAD

**Discussion by ezzra** [copier-org/copier#1858] [Q&A]
I want to create multiple files based on a list, but it looks like to me there is no option for that. If I use jinja like this for a file name:

```
{% for item in items %}{{ item }}.yaml{% endfor %}.jinja
```
then I will of course just get a nice filename like `item1.yamlitem2.yaml`

**Comment by sisp** [comment]
Duplicate of #1271. See also #1855 for a PR that adds this feature.
